frontends/web/src/routes/new-settings/components/singledropdown/singledropdown.module.css

```diff
@@ -0,0 +1,50 @@
+.dropdown {
+    appearance: none;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' fill='none' stroke='currentColor' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-labelledby='chevron down' color='%23777'%3E%3Cdefs/%3E%3Cpolyline points='6 10 12 16 18 10'/%3E%3C/svg%3E%0A");
```

Comment:
`currentColor` only works if the SVG elements are in the DOM (afaik), but not over CSS. Could you hardcode the color?

Reply:
It does work it seems like.. Without current colour it'll be not visible. Also tried changing it to red and the chevron changed to red, too:)

Reply:
yes red is a color, but currentColor is like CSS inherit, but that doens't work if the svg is in backrgound-imag data url, only if it is in the dom.

Reply:
Sorry, I understood what you meant. I was thinking the `stroke` attribute. But im not sure what you meant either by "doesn't work" here because it definitely inherits something, as leaving it blank will make the chevron invisible.
<img width="343" alt="Screenshot 2023-05-08 at 12 55 05" src="https://user-images.githubusercontent.com/28676406/236806576-4e4597a7-131a-4b04-82b8-c8af66f21d71.png">

Anyway, doesn't really matter i guess. But I went with your suggestion w/ hardcoding it with `#7777` to be safe. :)

Reply:
gray must be some kind of fallback, I made a quick and dirty test, inline svg vs. css background with svg
both use currentColor, the parent defines green color. the inline svg inherits green. the css background doesn't, as it can't I think as it doesn't know about the context. same with external svg. 

https://jsfiddle.net/2scndvbe/

Reply:
Oh haha, such a neat coincidence that it fallbacks to the exact gray we're using too! :) Thanks Eni 👍